website/docs/config/networks.mdx

```diff
@@ -3,16 +3,16 @@ title: Testnet Configuration
 ---
 
 :::note
-This document details the configuration and use of Tezos testnets. If you are looking for information on running or originating to a sandbox, please see the Flextesa documentation [here](../plugins/plugin-flextesa.mdx/)
+This document details the configuration and use of Tezos testnets. If you are looking for information on running or originating to a sandbox, please see the Flextesa documentation [here](docs/plugins/plugin-flextesa)
 :::
 
 ## Introduction
 
-Taqueria provides support to interact with Tezos networks in a simplified and streamlined manner. Taqueria provides a centralized configuration for testnets (including faucet keys) which can then be targeted by Taqueria commands through the use of Taqueria environments
+Taqueria provides support to interact with Tezos networks in a simplified and streamlined manner. Taqueria provides a centralized configuration for testnets (including faucet information) which can then be targeted by Taqueria commands through the use of Taqueria environments
 
 To work with Taqueria networks, these are the basic steps you need to take:
 1. Add a network configuration to the Taqueria `.taq/config.json` file
-2. Add a faucet key
+2. Add a faucet key (please see the faucets section [below](/docs/config/networks/#Faucets))
```

Comment:
I think this should be removed now and instead have reference the `taq root account`.